Source/Preview/App/Fuse.Simulator/SimulatorClient.uno

```diff
@@ -205,7 +203,7 @@ namespace Fuse.Simulator
 		{
 			_running = false;
 			
-			_alsoReceieveMessagesFromPipe.Dispose();
+			//_alsoReceieveMessagesFromPipe.Dispose();
```

Comment:
Just delete the line, no point in keeping it commented out...

Reply:
Also done - thank u for your review!